x-pack/plugin/ml/src/main/java/org/elasticsearch/xpack/ml/autoscaling/MlAutoscalingResourceTracker.java

```diff
@@ -110,19 +119,21 @@ public static void getMlAutoscalingStats(
             processorsAvailableFirstNode,
             maxOpenJobsPerNode,
             mlDummyAutoscalingEntity,
-            listener
+            listener,
+            clusterSettings.get(MachineLearning.ALLOCATED_PROCESSORS_SCALE)
         );
     }
 
     static void getMemoryAndProcessors(
         MlAutoscalingContext autoscalingContext,
         MlMemoryTracker mlMemoryTracker,
         Map<String, Long> nodeSizeByMlNode,
-        long perNodeAvailableModelMemoryInBytes,
+        long perNodeAvailableModelMemoryBytes,
         int perNodeAvailableProcessors,
         int maxOpenJobsPerNode,
         MlDummyAutoscalingEntity dummyAutoscalingEntity,
-        ActionListener<MlAutoscalingStats> listener
+        ActionListener<MlAutoscalingStats> listener,
+        int allocatedProcessorsScale
```

Comment:
I think it's common (and makes sense) to put the listener last. 

`allocatedProcessorsScale` sounds like it belongs with memory, process and the like